pkg/workspace/library_loader.go

```diff
@@ -46,30 +46,36 @@ func NewLibraryLoader(libraryCtx LibraryExecutionContext,
 	}
 }
 
-func (ll *LibraryLoader) Schemas() ([]*yamlmeta.Document, error) {
+func (ll *LibraryLoader) Schemas() (Schema, error) {
```

Comment:
if we are returning a single schema should this method be named Schema() as well?